### PR TITLE
chore: upgrade desktop app packages

### PR DESCRIPTION
**Pull request by ruisaraiva19**
- Remove `package-lock.json` in favor of `yarn.lock`
- Upgrade Husky and use `lint-staged` with it
- Upgrade `sass-loader` and migrate from `node-sass` to `sass` (`dart-sass`) as recommended in the `sass-loader` [v9.0.0 release](https://github.com/webpack-contrib/sass-loader/releases/tag/v9.0.0)

### CODE REVIEW

File: desktop-app/package.json [hunk 1]
```diff
@@ -40,24 +39,25 @@
...
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
   "lint-staged": {
     "*.{js,jsx}": [
       "cross-env NODE_ENV=development eslint --cache --format=pretty",
-      "prettier --ignore-path .eslintignore --single-quote --write",
-      "git add"
```
Comment (manojVivek) [inline desktop-app/package.json:47]: @ruisaraiva19 We don't need the `git add` any more?

Comment (ruisaraiva19) [reply desktop-app/package.json:47]: @manojVivek Yes, since `lint-staged` version `10.0.0`, it automatically includes `git add` as the final step.

**Comment by manojVivek** [comment]
@ruisaraiva19 Sorry for the delay in merging this. 🙇‍♂️ 
Doing it now and thanks for the package upgrades.

**Comment by manojVivek** [comment]
@all-contributors Please add @ruisaraiva19 for code.

**Comment by manojVivek** [comment]
@ruisaraiva19 

- Can you let us know your Twitter handle? We would love to give a shout out for your work from our twitter account.

- Please join our community on [Slack][slack], for any discussions.

Looking forward to more collaborations in the future! 💪🏻

[slack]: https://join.slack.com/t/responsively/shared_invite/zt-h0ouywhs-IIeHj39ymz5IOrgcOa_tRA

**Pull request merged**